form.js

```diff
@@ -22,137 +22,115 @@ define('form', ['doc'], function($) {
 		};
 	}
```

Comment:
Do we still need that? And perhaps we can remove the IE8 fallback, on line 10 and 165

https://developer.mozilla.org/pt-BR/docs/Web/JavaScript/Reference/Global_Objects/String/Trim

@tisvasconcelos

Reply:
We agree.